pkg/pillar/cmd/zedrouter/dnsmasq_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package zedrouter
+
+import (
+	"bytes"
+	"net"
+	"regexp"
+	"testing"
+
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/sirupsen/logrus"
+)
+
+func init() {
+	logger = logrus.StandardLogger()
+	log = base.NewSourceLogObject(logger, "zedrouter", 1234)
+}
+
+type dnsmasqConfigletParams struct {
+	ctx          *zedrouterContext
+	bridgeName   string
+	bridgeIPAddr string
+	netstatus    *types.NetworkInstanceStatus
+	hostsDir     string
+	ipsetHosts   []string
+	uplink       string
+	dnsServers   []net.IP
+	ntpServers   []net.IP
+}
+
+func newDnsmasqConfigletParams() dnsmasqConfigletParams {
```

Comment:
Using new might mislead, that is, it this is a method to add new params.  Starting the name with `test` or `dummy` might better express what the function does and where it fits in the scheme of things.

Just a nit, and a suggestion.

Reply:
@famleebob this is in *_test.go which is all about go unit tests.

Reply:
@famleebob  I renamed it to: `exampleDnsmasqConfigletParams`